jwst/coron/align_refs_step.py

```diff
@@ -38,6 +41,15 @@ def process(self, target, psf):
             # Open the input psf images
             psf_model = datamodels.open(psf)
 
+            # Retrieve the lenth of the smoothing box for the filter
+            box_size = self.smoothing_box_length
+
+            # Smooth the psf images
```

Comment:
Again, I would not refer to this as a smoothing operation. The bad pixel replacement values are being derived by applying a kind of smoothing operation to the images, but overall the resulting images are not being smoothed. We're simply replacing individual bad pixels. So "Replace bad pixels in the psf images" and "Replace bad pixels in the target images".